frontend/src/components/pipelines-or-deployments/pipelines/Pipelines.jsx

```diff
@@ -400,9 +434,17 @@ function Pipelines({ type }) {
           {record.last_run_status === "processing" ? (
             <SpinnerLoader />
           ) : (
-            <Typography.Text className="p-or-d-typography" strong>
-              {record?.last_run_status}
-            </Typography.Text>
+            <SpaceWrapper>
```

Comment:
@chandrasekharan-zipstack Can we align the status text and the reload button horizontally? Just use <Space> instead of <SpaceWrapper> and it should work fine.

Reply:
@tahierhussain This is done.